packages/auth-browser/src/lib/auth-browser.ts

```diff
@@ -11,6 +11,11 @@ import { checkAndSignEVMAuthMessage } from './chains/eth';
 import { checkAndSignSolAuthMessage } from './chains/sol';
 
 /**
+ * SUPPORTED CHAINS: EVM, Solana, Cosmos
+ *
+ * !! NOTE !!
+ * This function is purely used for crafting the authSig for access control conditions & decryption. For SessionSigs, you won't be able to use this
```

Comment:
It can also be used to pass in the authSig as jsParams or Eth Wallet AuthMethod for signSessionKey and claiming as well. So we still use authSig for these operations